README.md

```diff
@@ -154,6 +154,18 @@ Example production environment config file:
 #{Rails.root}/config/environments/production.yml
 ```
 
+### Extra sources
+
+You can load extra sources from the config folder during initialization by setting the `extra_sources` configuration option.
+
+```ruby
+Config.setup do |config|
+  config.extra_sources = ['extra_settings']
```

Comment:
Hi, 

It might be interesting to allow it to be any "source-like" implementation, rather than just a filename within the config root.

`config.add_source!` will then take care of either attaching it as a file source, or as a custom source.

Reply:
got it. will make the changes